inference/src/fixpoint.rs

```diff
@@ -115,23 +115,32 @@ impl FoundFixpoint {
     }
 }
 
-pub fn qalpha<O, L, B>(
-    infer_cfg: InferenceConfig,
-    conf: &SolverConf,
-    m: &Module,
-    print_invariant: bool,
-) where
+pub fn qalpha<O, L, B>(infer_cfg: InferenceConfig, m: &Module, print_invariant: bool)
+where
     O: OrderSubsumption<Base = B>,
     L: LemmaQf<Base = B>,
     B: Clone + Debug + Send,
 {
+    let solver_confs = [
+        SolverConf::new(SolverType::Z3, true, &infer_cfg.fname, 3, 1),
+        SolverConf::new(SolverType::Cvc5, true, &infer_cfg.fname, 3, 2),
+        SolverConf::new(SolverType::Z3, true, &infer_cfg.fname, 60, 3),
+        SolverConf::new(SolverType::Cvc5, true, &infer_cfg.fname, 60, 4),
+        SolverConf::new(SolverType::Z3, true, &infer_cfg.fname, 600, 5),
+        SolverConf::new(SolverType::Cvc5, true, &infer_cfg.fname, 600, 6),
+        SolverConf::new(SolverType::Z3, true, &infer_cfg.fname, 0, 7),
+    ];
+    let confs = solver_confs.iter().collect_vec();
+    let simulation_conf = SolverConf::new(SolverType::Z3, true, &infer_cfg.fname, 3, 0);
```

Comment:
Please move all of this policy configuration to a separate function (even if it just returns the constants for `solver_confs` and `simulation_conf`) and add some documentation on why you made these choices. It could also use a reminder as to what those two integer arguments to `SolverConf::new` are.